network-api/networkapi/wagtailpages/templates/wagtailpages/primary_page.html

```diff
@@ -16,17 +16,23 @@
 
   {% primary_page_menu page %}
 
-  <div class="container cms">
-    <div class="row my-1 d-flex align-items-center">
-      <div class="pb-5 {% if page.specific.narrowed_page_content %}col-lg-8 offset-lg-2{% else %}col-12{% endif %}">
-        {% for block in page.body %}
-          {% if block.block_type == 'heading' %}
-            <h1>{{ block.value }}</h1>
-          {% else %}
-            {% include_block block %}
-          {% endif %}
-        {% endfor %}
-      </div>
-    </div>
+  <div class="cms">
```

Comment:
Doing the same thing as campaign page, with the only difference here being the if/else logic for narrow pages on line 24